stack/globals.go

```diff
@@ -106,6 +106,23 @@ func (ge *globalsExpr) has(name string) bool {
 	return ok
 }
 
+func removeUnset(globals map[string]expression) {
```

Comment:
Did this instead of modeling unset as an actual value, like null. Since unset is very specific to undefining globals it seemed simpler to keep it constrained on its usage, instead of allowing it to be used as an value, like inside lists/objs/ternaries etc. But maybe that would be desireable, but then we need to think on what to do when we have unset inside an object and that object is rendered on code generation, etc.